node/node.go

```diff
@@ -1349,33 +1392,47 @@ func (n *Node) initInfoAPI() error {
 		return fmt.Errorf("problem creating proof of possession: %w", err)
 	}
 
-	service, err := info.NewService(
-		info.Parameters{
-			Version:   version.CurrentApp,
-			NodeID:    n.ID,
-			NodePOP:   pop,
-			NetworkID: n.Config.NetworkID,
-			VMManager: n.VMManager,
-			Upgrades:  n.Config.UpgradeConfig,
-
+	infoReceiver := &info.Info{
+		Parameters: info.Parameters{
+			Version:          version.CurrentApp,
+			NodeID:           n.ID,
+			NodePOP:          pop,
+			NetworkID:        n.Config.NetworkID,
+			VMManager:        n.VMManager,
+			Upgrades:         n.Config.UpgradeConfig,
 			TxFee:            n.Config.TxFee,
 			CreateAssetTxFee: n.Config.CreateAssetTxFee,
 		},
-		n.Log,
-		n.vdrs,
-		n.chainManager,
-		n.VMManager,
-		n.Config.NetworkConfig.MyIPPort,
-		n.Net,
-		n.benchlistManager,
-	)
+		Log:          n.Log,
+		Validators:   n.vdrs,
+		ChainManager: n.chainManager,
+		VMManager:    n.VMManager,
+		MyIP:         n.Config.NetworkConfig.MyIPPort,
+		Networking:   n.Net,
+		Benchlist:    n.benchlistManager,
+	}
+
+	service, err := info.NewService(infoReceiver)
 	if err != nil {
 		return err
 	}
+
+	infoPattern, infoConnHandler := infov1connect.NewInfoServiceHandler(
+		infoconnect.NewConnectInfoService(infoReceiver),
+	)
+	reflectionPattern, reflectionHandler := grpcreflect.NewHandlerV1(
+		grpcreflect.NewStaticReflector(infov1connect.InfoServiceName),
+	)
+
+	mux := http.NewServeMux()
+	mux.Handle(reflectionPattern, reflectionHandler)
+	mux.Handle(infoPattern, infoConnHandler)
+	n.APIServer.AddHeaderRoute("info", mux)
+
 	return n.APIServer.AddRoute(
 		service,
 		"info",
-		"",
+		"info",
```

Comment:
This looks like a bug from a bad merge/rebase, we probably need to undo this diff (there's a similar diff on the other apis when we call `AddRoute`).